lib/cosmos/system/system.rb

```diff
@@ -60,7 +60,7 @@ class System
     instance_attr_reader :limits_set
 
     # Known COSMOS ports
-    KNOWN_PORTS = ['CTS_API', 'TLMVIEWER_API', 'CTS_PREIDENTIFIED']
+    KNOWN_PORTS = ['CTS_API', 'TLMVIEWER_API', 'CTS_PREIDENTIFIED', 'CTS_CMD_ROUTER']
```

Comment:
I did a blog post on this: http://cosmosrb.com/news/2017/02/22/routers/ but sounds like it's due for an update!